steps/const.go

```diff
@@ -42,13 +42,17 @@ const (
 	CacheRestoreCocoapodsVersion = "1"
 	CacheRestoreCarthageID       = "restore-carthage-cache"
 	CacheRestoreCarthageVersion  = "1"
+	CacheRestoreNPMID            = "restore-npm-cache"
```

Comment:
Do we still use the old cache steps in the default workflows?

Reply:
Yes, I didn't touch the Flutter workflows yet. I'll clean up the old cache steps once I'm done with all platforms.